lldb/test/API/lang/swift/deployment_target/dlopen_module.m

```diff
@@ -0,0 +1,18 @@
+#include <Foundation/Foundation.h>
+#include <dlfcn.h>
+#include <libgen.h>
+
+@protocol FooProtocol
++ (void)f;
```

Comment:
Should this be `- (void)f;`? Here's `f` is a class method, but then below `f` is called on an instance.